afs/config_handler.py

```diff
@@ -125,16 +137,13 @@ def next_node(self, data=None, debug=False):
         :return: Response JSON
         :rtype: dict
         """
-        if data is not None:
-            if isinstance(data, DataFrame):
-                column_reverse_mapping = {v: k for k, v in self.get_column().items()}
-                data = data.rename(columns=column_reverse_mapping)
-                data = dict(data=data.to_dict())
-                return self.flow_obj.exe_next_node(data, debug=debug)
-            else:
-                raise AssertionError('Type error, data must be DataFrame type')
+        if isinstance(data, DataFrame):
+            column_reverse_mapping = {v: k for k, v in self.get_column().items()}
+            data = data.rename(columns=column_reverse_mapping)
+            data = dict(data=data.to_dict())
+            return self.flow_obj.exe_next_node(data, debug=debug)
         else:
-            raise AssertionError('Data cannot be none')
+            raise TypeError('Type error, data must be DataFrame type')
```

Comment:
Cause it will raise `TypeError`, the `Type error` in error message is a little bit duplicate.

Reply:
It is good to express briefly.